drivers/can/can_nxp_s32_canxl.c

```diff
@@ -179,7 +179,7 @@ static inline int can_nxp_s32_get_rx_fifo_filter(struct can_nxp_s32_data *data)
 			continue;
 		}
 
-		if ((data->rx_fifo[0].Header.Id & mask) ==
+		if ((data->rx_fifo[queue_idx].Header.Id & mask) ==
```

Comment:
is this backwards compatible with older sil cuts?

Reply:
I am afraid that, no.

Reply:
if this change is breaking support for users with older Sil cuts, then IMO it should be noted in RN.
On the other hand, `zephyr/soc/nxp/s32/s32ze/Kconfig.soc` defines only `SOC_PART_NUMBER_P32Z270ADCK0MJFT`. Does the new Sil version keeps the same part numbering or are we forgetting to update this?

Reply:
I will document in RN. For the new Sil, I haven't found the name of the new cut yet, so I propose to use `SOC_PART_NUMBER_P32Z270` only. What do you think, then I'll re-push